python/nano/src/bigdl/nano/tf/keras/inference_utils.py

```diff
@@ -39,12 +39,14 @@ def quantize(self,
                  max_trials: int = None,
                  batch=None,
                  inputs: List[str] = None,
-                 outputs: List[str] = None):
+                 outputs: List[str] = None,
+                 sample_size: int = 100):
         """
         Post-training quantization on a keras model.
 
-        :param calib_dataset:   A tf.data.Dataset object for calibration. Required for
-                                static quantization. It's also used as validation dataloader.
+        :param calib_dataset:   A tf.data.Dataset object for calibration without setting
+                                batch_size or batch_size=1. Required for static quantization.
+                                It's also used as validation dataloader.
         :param precision:       Global precision of quantized model,
                                 supported type: 'int8', 'bf16', 'fp16', defaults to 'int8'.
         :param accelerator:     Use accelerator 'None', 'onnxruntime', 'openvino', defaults to None.
```

Comment:
'onnxruntime' could be removed since we have not support it.

Reply:
we support it in `trace` through `tf-onnx`, while not in `quantize` for now.

Reply:
Can we support tf->onnx->inc->ort?

Reply:
We have supported tf onnxruntime quantize in another PR: #6461